src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -554,7 +554,7 @@ public enum CassandraRelevantProperties
     TCM_USE_NO_OP_REPLICATOR("cassandra.test.use_no_op_replicator", "false"),
 
     TEST_BBFAILHELPER_ENABLED("test.bbfailhelper.enabled"),
-    TEST_BLOB_SHARED_SEED("cassandra.test.blob.shared.seed"),
+    TEST_BLOB_SHARED_SEED("cassandra.test.blob.shared.seed", "42"),
```

Comment:
current logic used current time, which meant that tests were non-deterministic.... the seed *must* be set in order to be able to reproduce issues

Reply:
Not sure I follow. Is the problem that we just didn't output the seed in whatever failure message was produced?

Reply:
so, the tests use `qt()` and controls the seed for the test.  The blob logic has a giant blob that is shared by all tests, and uses a seed to populate the bytes... this seed is decoupled from the test seed, and would use `currentTimeMillis`; this caused the tests to be non-deterministic and not repeatable when errors were found.

Reply:
Yeah, not something I'd expect to unify in this patch.